netsim/ansible/templates/routing/eos.j2

```diff
@@ -8,3 +9,26 @@
 {{     routemap.common_route_map_entry(p_entry) }}
 {%-   endcall %}
 {% endif %}
+{% if routing._prefix.ipv4|default(False) %}
+!
+! IPv4 prefix lists
+!
+{%   for p_name,p_value in routing._prefix.ipv4.items() %}
```

Comment:
Isn't this a duplication, if the for loop is already performed by the included file `_prefix_list_ios.j2`?

(same for ipv6?)

Reply:
EOS has a different syntax for IPv6 prefix lists than for IPv4 ones, and once I figured that out, I decided to inline them both for consistency (and who knows what else might pop up later)